script/common.bash

```diff
@@ -34,18 +34,31 @@ get_go_version() {
   fi
 }
 
+# install_gvm
+# Install gvm to /usr/local/bin.
+# To read more about installing gvm in other platforms: https://github.com/andrewkroh/gvm#installation
+install_gvm() {
+  # Install gvm
+  if [ ! -f "/usr/local/bin/gvm" ]; then
+    curl -sL -o ~/bin/gvm https://github.com/andrewkroh/gvm/releases/download/v0.3.0/gvm-linux-amd64
+    chmod +x /usr/local/bin/gvm
+  fi
```

Comment:
`script/common.bash` gets run, indirectly, as part of our Makefile: https://github.com/elastic/apm-server/blob/73e63c04a5feaba3cfcbe60cd6173fdc517ea593/Makefile#L9

Thus, this is going to break devs: Linux because this needs root, and MacOS because this assumes Linux. Can we install it into ~/.gvm? I would also be much more comfortable if this checked a sha256sum.